utils/spacewalk-common-channels

```diff
@@ -164,6 +164,16 @@ def channel_get_details(client, channel_label):
     except xmlrpclib.Fault as e:
         return None
 
+def get_repo_list(client):
```

Comment:
Just a little nitpick. That's not really a list, is it?

Reply:
renamed :-)